### ISSUE TITLE
rpm core, build and sign brainsplit

### ISSUE THREAD

**Issue by kloczek** [rpm-software-management/rpm#777]
In https://github.com/rpm-software-management/rpm/commit/73a41da965337aaacabd53924f1b67b967e0476e is noted that separation rpm, rpmb and rpms python modules is causing some problems.
It would be good to know about those problems because now is not possible to have minimal set of packages without installed sign and build parts of the rpm because dnf uses rpm python module.
I would be glad to hear more about those problems because IMO it is worth to preserve that python separation.
If python module is connection all other rpm libraries separation on those libraries level does not make to much sense.

**Comment by ignatenkobrain** [comment]
If you want minimal set of packages, you should not have DNF in there since it pulls in Python which is not small.

Python bindings are bindings for all different RPM libraries so it just does not make sense to have some "fallbacks" in there.

**Comment by kloczek** [comment]
> If you want minimal set of packages, you should not have DNF in there since it pulls in Python which is not small.
Do you know any tool which can operate on dnf repositories without using rpm python binding?
> Python bindings are bindings for all different RPM libraries so it just does not make sense to have some "fallbacks" in there.
Who told this?
Raw rpm does not provide any functionality to operate on remote repositories.
You initiated merging those three modules so I'm expecting that you will be able to tell a bit more about  those "more and more issues over the years". I'm not aware of any of those issues and this is why I'm asking.
In other words I don't see proper/any justification of that kind of change.
